imports/plugins/included/default-theme/client/styles/staticPages.less

```diff
@@ -0,0 +1,17 @@
+.staticBody {
+    font-size: 18px;
+    line-height: 30px;
+}
+
+.staticHead {
+    font-size: 36px;
+}
+
+.pageManager {
+    margin-top: 4rem;
+}
```

Comment:
@OlatunjiYso , I think you should look into the naming convention for styles, you using pascalCase and snake-case, follow a convention a use it all through.